### PR TITLE
docs: update docs

### PR DESCRIPTION
**Pull request by mitosagi**
## Description

Update the documents with the latest information. I will write the English version after the Japanese version is accepted.

## Related Issue

- #131

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,51 +7,48 @@
...
-  <img src="./docs/images/screen1.png" width="160" />
-  <img src="./docs/images/screen2.png" width="320" />
+  <img src="./docs/images/tab1.png" width="160" />
+  <img src="./docs/images/tab2.png" width="160" />
+  <img src="./docs/images/tab1dark.png" width="160" />
+  <img src="./docs/images/tab2dark.png" width="160" />
 </p>
 
-AviUtl本体やプラグイン・スクリプトの導入を補助するソフトウェアです。
+AviUtlを手軽に導入できるソフトウエアです
```
Comment (hal-shu-sato) [inline README.md:16]: Which expression is better?
```suggestion
AviUtlを手軽に導入できるソフトウェアです
```

File: README.md [hunk 2]
```diff
@@ -7,51 +7,48 @@
...
+- AviUtl・拡張編集やプラグイン・スクリプトのインストール
 - 50を超えるプラグインと多数のスクリプトに対応
 - インストール済みのAviUtlへ追加可能
 
 _Read this in [English](./README.en.md)_.
 
 ## ダウンロード
 
-[Releases](https://github.com/hal-shu-sato/apm/releases/latest)のAssetsから`AviUtl.Package.Manager-{version}.Setup.exe`をダウンロードします。インストール不要のzip版もあります。
+[ダウンロードページ](https://github.com/hal-shu-sato/apm/releases/latest)のAssetsから`AviUtl.Package.Manager-{version}.Setup.exe`をダウンロードします。
```
Comment (hal-shu-sato) [inline README.md:26]: Let's be unified about punctuation.

```suggestion
[ダウンロードページ](https://github.com/hal-shu-sato/apm/releases/latest)のAssetsから`AviUtl.Package.Manager-{version}.Setup.exe`をダウンロードします
```

File: docs/index.md [hunk 3]
```diff
@@ -15,37 +17,32 @@
 
 ### ダウンロード
 
-[Releases](https://github.com/hal-shu-sato/apm/releases/latest)のAssetsから`AviUtl.Package.Manager-{version}.Setup.exe`をダウンロードします。インストール不要のzip版もあります。
+[ダウンロードページ](https://github.com/hal-shu-sato/apm/releases/latest)のAssetsから`AviUtl.Package.Manager-{version}.Setup.exe`をダウンロードします。
```
Comment (hal-shu-sato) [inline docs/index.md:20]: Let's be unified about punctuation.

```suggestion
[ダウンロードページ](https://github.com/hal-shu-sato/apm/releases/latest)のAssetsから`AviUtl.Package.Manager-{version}.Setup.exe`をダウンロードします
```

**Review by hal-shu-sato — COMMENTED** [review]
Good!
I have some points to check.

**Comment by hal-shu-sato** [comment]
You can either punctuate or not, as long as it is unified.

File: README.md (hunk 1 above)
Comment (mitosagi) [reply README.md:16]: Suggested change looks the same. Either way, I'm having a very hard time with the wording here, so if you have a suggestion, I'd like to use it.

Comment (hal-shu-sato) [reply README.md:16]: NHK seems to approve of both.
https://www.nhk.or.jp/bunken/research/kotoba/pdf/20150401_5.pdf

Wikipedia has ソフトウェア page.
https://ja.wikipedia.org/wiki/%E3%82%BD%E3%83%95%E3%83%88%E3%82%A6%E3%82%A7%E3%82%A2

Google and Yahoo! JAPAN has more ソフトウェア results than ソフトウエア.

I think ソフトウェア is more general.

**Review by hal-shu-sato — APPROVED** [review]
Good revision!

**Pull request merged**